src/vuex/plugins/session-storage.js

```diff
@@ -23,19 +23,22 @@ export const sessionStoragePlugin = store => {
           wallet: savedStore.wallet,
           kyc: savedStore.kyc,
           keyValue: savedStore.keyValue,
-          session: savedStore.session,
           idleHandler: savedStore.idleHandler,
         })
 
         break
       }
       default:
+        if (mutation.type === vuexTypes.UPDATE_LOGOUT_AT) {
```

Comment:
Let's try to make the first condition also in a constant, and combine it into one condition